projects/plugins/social/src/js/components/admin-page/index.jsx

```diff
@@ -141,9 +95,9 @@ const ConnectionSection = () => {
 	const { apiNonce, apiRoot, registrationNonce } = window.jetpackSocialInitialState;
 	return (
 		<ConnectScreenRequiredPlan
-			buttonLabel={ __( 'Get Jetpack Social', 'jetpack-social' ) }
-			priceAfter={ 4.5 }
-			priceBefore={ 9 }
+			buttonLabel={ __( 'Connect Jetpack Social', 'jetpack-social' ) }
```

Comment:
Do you think there's a `ConnectScreen` component we could use, so tha we don't have a required plan and the price props?

Reply:
Possibly, I didn't dive too deep into it. I guess it could be [this one](https://github.com/Automattic/jetpack/blob/412db5a8829c99e4ceea06c4613fffe702ce9930/projects/js-packages/connection/components/connect-screen/basic/index.jsx), will give it a try. 👍🏼